reyna/src/com/b2msolutions/reyna/Dispatcher.java

```diff
@@ -108,8 +119,12 @@ public static Result canSend(Context context) {
         return Result.OK;
     }
 
-    private static boolean canSendNow(BlackoutTime blackoutTime, String window) throws ParseException {
-        return blackoutTime.canSendAtTime(new GregorianCalendar(), window);
+    private static boolean canSendNow(BlackoutTime blackoutTime, String window, GregorianCalendar now) throws ParseException {
+        return blackoutTime.canSendAtTime(now, window);
+    }
+
+    private static boolean canSendNow(long startUtc, long endUtc, long nowUtc) {
```

Comment:
Can we have a different function name. like canSendBetween or any other name.
it confused me at the start with the refactor to previous canSendNow